src/components/integrated/OakLessonNavItem/OakLessonNavItem.tsx

```diff
@@ -143,6 +150,7 @@ export const OakLessonNavItem = <C extends ElementType = "a">(
       $borderColor={borderColor}
       $ba="border-solid-l"
       $disabled={disabled}
+      $color="text-primary"
```

Comment:
I decided to fix the issue in the component to avoid breaking encapsulation any further by making the `color` explicit. Another option was to add to the reset styles in both OWA and oak-components — but this could be a rug-pull and cause subtle defects elsewhere. It would also make the component less portable since it relies on an untraceable styling dependency.